subsys/bluetooth/shell/mcc.c

```diff
@@ -575,7 +575,7 @@ int cmd_mcc_init(const struct shell *shell, size_t argc, char **argv)
 	cb.icon_url_read     = &mcc_icon_url_read_cb;
 	cb.track_changed_ntf = &mcc_track_changed_ntf_cb;
 	cb.track_title_read  = &mcc_track_title_read_cb;
-	cb.track_dur_read    = &mcc_track_dur_read_cb;
+	cb.track_duration_read    = &mcc_track_duration_read_cb;
```

Comment:
Bad indentation

Reply:
Also think & should be removed, because meaningless.

Reply:
Fixed indentation

Reply:
It is not meaningless, but it is optional and not required.
Removed.